apps/docs/pages/guides/auth/social-login/auth-linkedin.mdx

```diff
@@ -10,6 +10,12 @@ To enable LinkedIn Auth for your project, you need to set up a LinkedIn OAuth ap
 
 ## Overview
 
+<Admonition type="caution" label="update">
```

Comment:
```suggestion
<Admonition type="caution" label="LinkedIn">
```

Reply:
keeping this as update because it looks weird if the label as "linkedin"
![image](https://github.com/supabase/supabase/assets/28647601/add698c7-9ebe-4b3d-8775-87dea4e14393)